Taskfile.deno.yaml

```diff
@@ -0,0 +1,29 @@
+version: 3
+
+vars:
+  NBL_DENO_VERSION: 1.37-dev
```

Comment:
Should this just be `1.37` since the tag is set to `dev` by default for these Taskfile builds?